src/root/filename.d

```diff
@@ -689,6 +704,6 @@ struct FileName
 
     extern (C++) const(char)* toChars() const
     {
-        return cast(char*)str; // toChars() should really be const
+        return str;
```

Comment:
I know this is trivial, but please don't mix code changes into commits marked as documentation only changes.